### ISSUE TITLE
Training for custom data

### ISSUE THREAD

**Issue by deepkshikha** [adeshpande3/LSTM-Sentiment-Analysis#36]
Can it be applied for 3 different class, Not the one provided??

**Comment by adeshpande3** [comment]
Yeah, you'll just have to change the encoding of the labels (from 1x2 dimensionality to 1x3) and you'll also have to change the numClasses variable, but I think that should be it.

**Comment by deepkshikha** [comment]
Thanks I trained by changing labels and class but the accuracy is very low my dataset is biased. If is there any suggestions.

**Comment by adeshpande3** [comment]
It's hard to pinpoint the exact reasons, but you should probably try experimenting with more complex/simple models, increasing the size and variety of your dataset (some sort of data augmentation), training for longer periods of time, using a different architecture, etc. 

One thing that I've found helpful is to use the smallest and simplest network you can, see if anything gets learned through training, and then slowly increase the size and complexity of the model until you see differences in accuracy.
